js/widgets/fixedToolbar.js

```diff
@@ -122,9 +122,8 @@ define( [ "jquery", "../widget", "../core", "../animationComplete", "../navigati
 			}
 		},
 
-		_handlePageBeforeHide: function( e, ui ) {
-			var o = this.options,
-				thisFooter, thisHeader, nextFooter, nextHeader;
+		_handlePageBeforeHide: function() {
+			var o = this.options;
```

Comment:
This servers no real point any more you only check it once and its not valid according to the style guide.